### PR TITLE
Add nginx reloading when new ssl released 

### PR DESCRIPTION
**Pull request by semeniak97mf**
Added reloading of the nginx automatically if a new ssl from certbot had been released.

[UPD] Actions: https://github.com/Atleta-network/blockscout/actions/runs/10686050439

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -13,13 +13,19 @@ jobs:
...
+      - name: Create Image Tarball for explorer-frontend
         run: |
           docker build --build-arg GIT_COMMIT_SHA=$(git rev-parse --short HEAD) --build-arg GIT_TAG=$(git describe --tags --abbrev=0) -t explorer-frontend .
           docker save -o explorer-frontend.tar explorer-frontend
           mkdir -p artifacts
           mv explorer-frontend.tar artifacts/
           
+      - name: Create Image Tarball for nginx
+        run: |
+          docker build --build-arg GIT_COMMIT_SHA=$(git rev-parse --short HEAD) --build-arg GIT_TAG=$(git describe --tags --abbrev=0) -t nginx-explorer nginx-docker/
```
Comment (hrls) [inline .github/workflows/deploy.yml]: Remove unused build args.

Comment (hrls) [reply .github/workflows/deploy.yml]: Move `nginx-docker/` into `docker/nginx/` subdir.

File: nginx-docker/docker-entrypoint.sh
Comment (hrls) [inline nginx-docker/docker-entrypoint.sh:1]: `command: /check_ssl.sh & ./entrypoint.sh "nginx -g 'daemon off;'"` or something like this, then we can remove thirdparty entrypoint script.

**Review by hrls — CHANGES_REQUESTED** [review]

File: deploy/services/certbot.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ version: '3.9'
 services:
   certbot:
     image: certbot/certbot
+    restart: unless-stopped
```
Comment (ales-tsurko) [inline deploy/services/certbot.yml]: Make sure it should be restarted. Probably its purpose is running once to just generate certificates. I don’t remember though.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (semeniak97mf) [reply .github/workflows/deploy.yml]: Done

File: nginx-docker/docker-entrypoint.sh
Comment (semeniak97mf) [reply nginx-docker/docker-entrypoint.sh:1]: Moved it to Dockerfile. 
This is a working and more optimal way, thank you

File: deploy/services/certbot.yml (hunk 2 above)
Comment (semeniak97mf) [reply deploy/services/certbot.yml]: Just in case, I changed this option to `always`. Thanks

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -43,7 +46,8 @@ jobs:
           key: ${{ secrets.SSH_PRIVATE_KEY }}
           script: |
             cd /blockscout
-            docker load -i explorer-frontend.tar
+            docker load --input explorer-frontend.tar
+            docker load --input nginx-explorer.tar
             docker compose down
             docker compose pull
-            docker compose up -d --build --remove-orphans
+            docker compose up --detach --build --remove-orphans
```
Comment (hrls) [inline .github/workflows/deploy.yml:53]: This `--build` flag is still required?

**Review by hrls — APPROVED** [review]

**Review by ales-tsurko — APPROVED** [review]

**Pull request merged**